pinc/misc.inc

```diff
@@ -464,6 +464,8 @@ function get_bool_param(array $arr, string $key, ?bool $default = null, bool $al
                 return true;
             } elseif ($s === "false" || $s === "0") {
                 return false;
+            } elseif ($s === "" && $allownull) {
```

Comment:
This change is required to allow passing in a null boolean from an HTML control.